### PR TITLE
21 refresh feature

### PR DESCRIPTION
**Pull request by lcwalther**

### CODE REVIEW

File: src/app/GlobalStyle.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default createGlobalStyle`
   body {
     font-family: 'Roboto', sans-serif;
     margin: 0;
-    background: rgb(250, 250, 250);
+    background: $primary;
```
Comment (FSchubert92) [inline src/app/GlobalStyle.js]: Wie hast du das geamcht? Das will ich ja auch!

File: src/app/GlobalStyle.js [hunk 2]
```diff
@@ -68,23 +68,11 @@ export default createGlobalStyle`
...
-  display:block;
-  width:100%;
-  border:none;
-  border-bottom:1px solid #E0E0E0;
-  background: none;
-  ::placeholder { color: #9E9E9E; font-size: 16px; }
-}
-input:focus 		{ outline:none; }
-
 /* LABEL ======================================= */
```
Comment (FSchubert92) [inline src/app/GlobalStyle.js]: Hier ist noch ein Kommentar drin

File: src/app/GlobalStyle.js [hunk 3]
```diff
@@ -125,6 +113,10 @@ input:focus ~ .bar:before, input:focus ~ .bar:after {
   width:50%;
 }
 
+textarea:focus ~ .bar:before, textarea:focus ~ .bar:after {
+  width:50%;
+}
+
 /* HIGHLIGHTER ================================== */
```
Comment (FSchubert92) [inline src/app/GlobalStyle.js]: Here as well ;)

File: src/cards/detailpage/CardDetailPage.js [hunk 4]
```diff
@@ -212,9 +216,11 @@ export default function CardsDetailPage(props) {
               />
             </EditButton>
           </ButtonList>
-          <StyledTitle>{card.title}</StyledTitle>
-          {card.tags && <TagList>{card.tags.map(renderTag)}</TagList>}
-          <StyledNotes>{card.notes}</StyledNotes>
+          <ContentGrid>
```
Comment (FSchubert92) [inline src/cards/detailpage/CardDetailPage.js]: Nice! Everything put in a grid!

File: src/cards/detailpage/CardEditForm.js [hunk 5]
```diff
@@ -21,6 +45,10 @@ const StyledButton = styled.button`
 
 const StyledTextArea = styled.textarea`
   width: 100%;
+  font-size: 16px;
+  height: 84px;
```
Comment (FSchubert92) [inline src/cards/detailpage/CardEditForm.js]: maybe it's better to make the more general things to the top? Like height at the beginning?

File: src/cards/detailpage/CardEditForm.js [hunk 6]
```diff
@@ -52,35 +80,52 @@ export default function CardForm({ card, onSubmit, setIsEditable }) {
     setIsEditable(false)
   }
 
+  const input = useRef(null)
+  useEffect(() => {
+    input.current.focus()
```
Comment (FSchubert92) [inline src/cards/detailpage/CardEditForm.js]: That's something i don't really understand

File: src/cards/detailpage/RefreshSlider.js [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+  }
+
+  useEffect(() => {
+    console.log('test')
+  })
+  return (
+    <React.Fragment>
+      Move video automatically to refresh queue in{' '}
+      <span style={{ fontWeight: 'bold', color: '#00cca9' }}>
+        {firstRender ? checkIfRefreshDate() : daysBeforeRefresh}
```
Comment (FSchubert92) [inline src/cards/detailpage/RefreshSlider.js]: if what? :)

File: src/common/Input.js [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+export default function Input({ placeholder, icon }) {
+  return (
+    <Group className="group">
+      <StyledInput className="input" />
+      <Bar className="bar" />
+    </Group>
+  )
+}
+// /* LABEL ======================================= */
```
Comment (FSchubert92) [inline src/common/Input.js]: Here is also still a lot of comments :)

File: src/cards/detailpage/CardEditForm.js (hunk 6 above)
Comment (lcwalther) [reply src/cards/detailpage/CardEditForm.js]: This is for giving the input field with the ref="input" an automatic focus on page load

File: src/cards/detailpage/CardEditForm.js (hunk 5 above)
Comment (lcwalther) [reply src/cards/detailpage/CardEditForm.js]: Good point!

**Pull request merged**